war/src/main/java/io/lumeer/engine/controller/CollectionMetadataFacade.java

```diff
@@ -532,80 +424,52 @@ public String getInternalCollectionName(String originalCollectionName) throws Us
-      DataDocument nameDocument = nameInfo.get(0);
-      String id = nameDocument.getId();
-
-      DataDocument metadataDocument = new DataDocument(LumeerConst.Collection.COLLECTION_REAL_NAME_KEY, collectionOriginalName);
-      dataStorage.updateDocument(collectionMetadataCollectionName(collectionInternalName), metadataDocument, id);
+      DataDocument collectionInfo = getCollectionMetadataDocument(collectionInternalName);
+      String documentId = collectionInfo.getId();
+      dataStorage.updateDocument(
+            METADATA_COLLECTION,
+            new DataDocument(
+                  LumeerConst.Collection.REAL_NAME_KEY,
+                  collectionOriginalName),
+            documentId);
 
       changeCollectionNameEvent.fire(new ChangeCollectionName(collectionOriginalName, collectionInternalName));
-
-      return true;
    }
 
    /**
-    * Reads current value of collection lock
+    * Reads time of last collection usage
     *
     * @param collectionName
     *       internal collection name
-    * @return String representation of the time of the last update of collection lock, empty string if lock is not set
-    * @throws CollectionNotFoundException
-    *       when metadata collection is not found
-    * @throws CollectionMetadataDocumentNotFoundException
-    *       when document in metadata collection is not found
+    * @return String representation of the time
     */
-   public String getCollectionLockTime(String collectionName) throws CollectionNotFoundException, CollectionMetadataDocumentNotFoundException {
-      checkIfMetadataCollectionExists(collectionMetadataCollectionName(collectionName));
-
-      List<DataDocument> lockInfo = dataStorage.run(queryDocumentFromCollectionMetadata(collectionName, LumeerConst.Collection.COLLECTION_LOCK_META_TYPE_VALUE));
-
-      if (lockInfo.isEmpty()) {
-         throw new CollectionMetadataDocumentNotFoundException(ErrorMessageBuilder.collectionMetadataNotFoundString(collectionName, LumeerConst.Collection.COLLECTION_LOCK_META_TYPE_VALUE));
-      }
-
-      DataDocument nameDocument = lockInfo.get(0);
-      return nameDocument.getString(LumeerConst.Collection.COLLECTION_LOCK_UPDATED_KEY);
+   public String getLastTimeUsed(String collectionName) {
+      return getCollectionMetadata(collectionName).getLastTimeUsed();
    }
 
    /**
-    * Sets collection lock to new value
+    * Sets the time of last collection usage to current time
     *
     * @param collectionName
     *       internal collection name
-    * @param newTime
-    *       String representation of the time of the last update of collection lock
-    * @return true if set was successful, false if time format is wrong or metadata collection does not exist
     */
-   public boolean setCollectionLockTime(String collectionName, String newTime) {
```

Comment:
They were never used and I thought that they were actually replaced by `setLastTimeUsedNow` and `getLastTimeUsed`.